tests/geometry/projection.rs

```diff
@@ -35,11 +35,11 @@ fn perspective_matrix_point_transformation() {
 
 #[cfg(feature = "arbitrary")]
 mod quickcheck_tests {
-    use na::{Orthographic3, Perspective3, Point3};
+    use na::{OpenGL, Orthographic3, Perspective3, Point3};
 
     quickcheck! {
         fn perspective_project_unproject(pt: Point3<f64>) -> bool {
-            let proj = Perspective3::new(800.0 / 600.0, 3.14 / 2.0, 1.0, 1000.0);
+            let proj = Perspective3::<OpenGL, f64>::new(800.0 / 600.0, 3.14 / 2.0, 1.0, 1000.0);
```

Comment:
another example of severe breakage.
Here f64 is not inferred properly and defaults to f32 is not specified.

I don't even understand how it worked before...